apps/api/src/controllers/player.ts

```diff
@@ -2,15 +2,17 @@ import { Request, Response } from 'express';
 
 export default {
   GET_fetchAllPlayers: async (req: Request, res: Response) => {
-    const players = await req.ctx.modelFactory.player.fetchAll(req.ctx);
+    const { page, pageSize } = req.query;
+    const pageNumber = parseInt(page as string) || 1;
+    const pageSizeNumber = parseInt(pageSize as string) || 100;
```

Comment:
```suggestion
    const pageSizeNumber = max(min(parseInt(pageSize as string) || 100, 100), 1);
```

> At least 1, not more than 100

How about we add some initial protection against using huge values on the backend?